packages/vendor/webpack.config.js

```diff
@@ -8,11 +8,13 @@ const { version } = require('./package.json');
 const createVendorEntry = require('./lib/createVendorEntry');
 const WebpackExportForemanVendorPlugin = require('./lib/WebpackExportForemanVendorPlugin');
 
-const filename = `[name].bundle-v${version}-[hash]`;
+const [, webpackMode = 'production'] = process.argv
+  .find(arg => arg.startsWith('--mode='))
+  .split('=');
```

Comment:
Why not use `NODE_ENV` to decide which one instead of parsing argv? I'm assuming the libraries that are problematic also check that to decide which version to use?

Reply:
`--mode=production/development` supported by webpack itself, by there documentation it will provide the relevant `NODE_ENV` based on that.
Notice before, it used to be hardcoded in the config.